src/components/pages/roster-page/roster-page.tsx

```diff
@@ -223,24 +458,35 @@ export const RosterPage = () => {
           </Table>
         </TableContainer>
       </Container>
-      <Dialog
-        open={alert.open}
-        onClose={handleAlertClose}
-        aria-labelledby="alert-dialog-title"
-        aria-describedby="alert-dialog-description"
-      >
-        <DialogTitle id="alert-dialog-title">{alert.title}</DialogTitle>
-        <DialogContent>
-          <DialogContentText id="alert-dialog-description">
-            {alert.message}
-          </DialogContentText>
-        </DialogContent>
-        <DialogActions>
-          <Button onClick={handleAlertClose} autoFocus>
-            OK
-          </Button>
-        </DialogActions>
-      </Dialog>
+      {deleteRosterEntryDialogOpen && (
+        <Dialog
+          open={deleteRosterEntryDialogOpen}
+          onClose={cancelDeleteRosterEntryDialog}
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
+        >
+          <DialogTitle id="alert-dialog-title">Remove User</DialogTitle>
```

Comment:
Maybe change this to `Remove Individual` since they aren't necessarily a user of the system.

Reply:
Yeah that makes more sense. Changed.